plotly/graph_objs/graph_objs_tools.py

```diff
@@ -1,190 +1,84 @@
-         "force_clean()"]) + "\n\n"
-    # Add key meta to doc
-    keymeta = info['keymeta']
-    if len(keymeta):
-        doc += "Valid keys:\n\n"
-        # Add each key one-by-one and format
-        width1 = LINE_SIZE-TAB_SIZE
-        width2 = LINE_SIZE-2*TAB_SIZE
-        width3 = LINE_SIZE-3*TAB_SIZE
-        undocumented = "Aw, snap! Undocumented!"
-        for key in keymeta:
-            # main portion of documentation
-            try:
-                required = str(keymeta[key]['required'])
-            except KeyError:
-                required = undocumented
-            try:
-                typ = str(keymeta[key]['key_type'])
-            except KeyError:
-                typ = undocumented
-            try:
-                val_types = str(keymeta[key]['val_types'])
-                if typ == 'object':
-                    val_types = ("{0} object | ".format(KEY_TO_NAME[key]) +
-                                 val_types)
-            except KeyError:
-                val_types = undocumented
-            try:
-                descr = str(keymeta[key]['description'])
-            except KeyError:
-                descr = undocumented
-            str_1 = "{0} [required={1}] (value={2})".format(
-                key, required, val_types)
-            if "streamable" in keymeta[key] and keymeta[key]["streamable"]:
-                str_1 += " (streamable)"
-            str_1 += ":\n"
-            str_1 = "\t" + "\n\t".join(textwrap.wrap(str_1,
-                                                     width=width1)) + "\n"
-            str_2 = "\t\t" + "\n\t\t".join(textwrap.wrap(descr,
-                                           width=width2)) + "\n"
-            doc += str_1 + str_2
-            # if a user can run help on this value, tell them!
-            if typ == "object":
-                doc += "\n\t\tFor more, run `help(plotly.graph_objs.{0" \
-                       "})`\n".format(KEY_TO_NAME[key])
-            # if example usage exists, tell them!
-            try:
-                if len(keymeta[key]['examples']):
-                    ex = "\n\t\tExamples:\n" + "\t\t\t"
-                    ex += "\n\t\t\t".join(
-                        textwrap.wrap(' | '.join(keymeta[key]['examples']),
-                                      width=width3)) + "\n"
-                    doc += ex
-            except:
-                pass
-            doc += '\n'
+    items_string = '\n\t* {}\n'.format('\n\t* '.join(items_classes))
+    doc += 'Valid Item Classes:\n{}\n'.format(items_string)
     return doc.expandtabs(TAB_SIZE)
 
 
-def update_keys(keys):
-    """Change keys we used to support to their new equivalent."""
-    updated_keys = list()
-    for key in keys:
-        if key in translations:
-            updated_keys += [translations[key]]
-        else:
-            updated_keys += [key]
-    return updated_keys
+def _make_dict_doc(name):
 
-translations = dict(
-    scl="colorscale",
```

Comment:
also @theengineear these deprecated keys will be part of the plot-schema. PR today.